### PR TITLE
Dls 5908 storage premise

### PR DESCRIPTION
**Pull request by priyankadhiman3006**
ticket details -> https://jira.tools.tax.service.gov.uk/browse/DLS-5908
Screenshot 2022-09-21 at 15 45 48
Screenshot 2022-09-21 at 15 46 04

### CODE REVIEW

File: app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html [hunk 1]
```diff
@@ -27,4 +27,4 @@
     }
 }
 
-<h1 @{headingClasses}>@{params.heading}</h1>
\ No newline at end of file
+<h1 class="govuk-heading-l" @{headingClasses}>@{params.heading}</h1>
```
Comment (adamliptrot-oc) [inline app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html:30]: Won't this potentially add two `class` attributes?

File: app/uk/gov/hmrc/fhregistrationfrontend/connectors/EmailVerificationConnector.scala [hunk 2]
```diff
@@ -86,6 +86,6 @@ class DefaultEmailVerificationConnector @Inject()(
     }
 
     val url = s"$emailVerificationBaseUrl/verification-requests"
-    http.POST[EmailVerificationRequest, Boolean](url, request)
+    http.POST[EmailVerificationRequest, Boolean](url, request).map(_ => true)
```
Comment (ginnyamazed) [inline app/uk/gov/hmrc/fhregistrationfrontend/connectors/EmailVerificationConnector.scala:89]: This page should not have been included, correct?  This change should be for local branch only?

File: app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html (hunk 1 above)
Comment (ginnyamazed) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html:30]: could you please help me understand why this page is needed?  Isn't the page heading covered within the play ui components and upgrade?

File: app/uk/gov/hmrc/fhregistrationfrontend/connectors/EmailVerificationConnector.scala (hunk 2 above)
Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/fhregistrationfrontend/connectors/EmailVerificationConnector.scala:89]: oops, missed it

File: app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html (hunk 1 above)
Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html:30]: @adamliptrot-oc  o yes, I think we shouldn't even use this Pageheading anymore

Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/PageHeading.scala.html]: realised not even using it, so reverted back

File: app/uk/gov/hmrc/fhregistrationfrontend/connectors/EmailVerificationConnector.scala (hunk 2 above)
Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/fhregistrationfrontend/connectors/EmailVerificationConnector.scala]: reverted

**Review by ginnyamazed — COMMENTED** [review]
is the page heading helper needed?  If it's 1 line of code, isn't it easier to follow what the code is doing by using the govuk class structure?

**Pull request merged**